### ISSUE TITLE
Use selected Deployment's `projectDir` to list files and include/exclude

### ISSUE THREAD

**Issue by dotNomad** [posit-dev/publisher#1854]
On hold for #1844 and #1851 

The `Project Files` section in the Home View will need to utilize the `parentDir` of the selected Deployment for all of the calls to `GET and POST /api/configurations/$name/files` to correctly list, include, and exclude files.

Blocked by #1950 - Bug

---

Part of the #1849 epic
